static/includes/apps/CustomAppIntro.md

```diff
@@ -6,3 +6,16 @@ Generally, any container that follows the [Open Container Initiative](https://op
 To deploy a custom application, go to **Apps** and click **Discover Apps**.
 Click **Custom App** to open the **[Install iX App]({{< relref "installcustomappscreens.md #install-ix-app-screen" >}})** screen with a guided installation wizard.
 Click <i class="material-icons" aria-hidden="true" title="more_vert">more_vert</i> > **Install via YAML** to open the **[Custom App]({{< relref "installcustomappscreens.md #custom-app-screen" >}})** screen with an advanced YAML editor for deploying apps using Docker Compose.
+
+TrueNAS also allows users to convert already installed apps into custom applications. 
+Click on the app to wish to convert, and then click the widget next to **Edit** to open the menu containing **Update** and **Convert to custom app** options.
```

Comment:
Is this the Edit button on the Applications Info widget? if it is, say that. Users might be confused but this sentence. How about this, if it is a correct interpretation of what you are saying here:
```suggestion
Select the row for the app to wish to convert, and then click **Edit** on the **Applications Info** widget.
The dropdown list with **Update** and **Convert to custom app** shows.
```